src/extensions/message.js

```diff
@@ -283,7 +283,7 @@ module.exports = Structures.extend('Message', Message => {
 				}
 			}
 
-			content = resolveString(content);
+			content = verifyString(this.options.content, RangeError, 'MESSAGE_CONTENT_TYPE', false);
```

Comment:
This is using the regular RangeError class, so it will result in `RangeError: MESSAGE_CONTENT_TYPE` instead of the wanted message

Reply:
Good point... but the RangeError isn't exported from discord.js. Might just switch this back to defaults and directly replace.